pymc/sampling_jax.py

```diff
@@ -134,13 +134,10 @@ def _sample_stats_to_xarray(posterior):
 
 def _get_log_likelihood(model: Model, samples, backend=None) -> Dict:
     """Compute log-likelihood for all observations"""
-    data = {}
-    for v in model.observed_RVs:
-        v_elemwise_logp = model.logp(v, sum=False)
-        jax_fn = get_jaxified_graph(inputs=model.value_vars, outputs=v_elemwise_logp)
-        result = jax.jit(jax.vmap(jax.vmap(jax_fn)), backend=backend)(*samples)[0]
-        data[v.name] = result
-    return data
+    elemwise_logpt = [model.logp(v, sum=False)[0] for v in model.observed_RVs]
+    jax_fn = get_jaxified_graph(inputs=model.value_vars, outputs=elemwise_logpt)
```

Comment:
We got rid of the `t` suffix just recently
```suggestion
    jax_fn = get_jaxified_graph(inputs=model.value_vars, outputs=elemwise_logp)
```